lib/generate-report.js

```diff
@@ -240,6 +240,10 @@ function generateReport(options) {
         scenario.time = formatDuration(scenario.duration)
       }
 
+      if (scenario.hasOwnProperty('description') && scenario.description) {
```

Comment:
As I've seem `scenario.description` will always be a string, so the `if(scenario.description)` should be sufficient, or are there any other cases I'm not seeing?

Reply:
"description" is not mandatory on json structure

Reply:
ok?